multus/multus.go

```diff
@@ -144,11 +143,51 @@ func validateIfName(nsname string, ifname string) error {
 	return err
 }
 
+func confAdd(rt *libcni.RuntimeConf, rawnetconflist []byte, binDir string, exec invoke.Exec) (cnitypes.Result, error) {
+	logging.Debugf("conflistAdd: %v, %s, %s", rt, string(rawnetconflist), binDir)
+	// In part, adapted from K8s pkg/kubelet/dockershim/network/cni/cni.go
+	binDirs := filepath.SplitList(os.Getenv("CNI_PATH"))
+	binDirs = append([]string{binDir}, binDirs...)
+	cniNet := libcni.NewCNIConfig(binDirs, exec)
+
+	confList, err := libcni.ConfFromBytes(rawnetconflist)
```

Comment:
The names `confList` and `rawnetconflist` in this context maybe misleading? I guess we are dealing with a single `NetworkConfig` not `NetworkConfigList`, am I correct?

Reply:
thank you for your comment! In this context, rawNetconf/conf (remove 'List') could be appropriate, so it is changed.